Add team-time extension

l-mbert opened the pull request:
## Description

<!-- A summary of your change. If you add a new extension or command, explain what it does. -->

## Screencast

<!-- If you add a new extension or command, include a screencast (or screenshot for straightforward changes). A good screencast will make the review much faster - especially if your extension requires registration in other services.  -->

## Checklist

- [ ] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [ ] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [ ] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [ ] I checked that files in the `assets` folder are used by the extension itself
- [ ] I checked that assets used by the `README` are placed outside of the `metadata` folder

pernielsentikaer commented on extensions/team-time/src/index.tsx:
```diff
@@ -0,0 +1,252 @@
+            )}
+          </ActionPanel.Section>
+          <ActionPanel.Section title="Team Time">
+            <Action.Push title="Add New Time" icon={Icon.AddPerson} target={<AddTime onAdd={revalidate} />} />
```
Wording is a bit confusing here, also maybe add ⌘+n for new?

l-mbert replied:
I have renamed it to "Add Teammember" and added the Shortcut! Thanks for the suggestion

pernielsentikaer reviewed (changes requested):
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension, and I have some feedback ready for you:

* You can edit a person and set the name blank

* Is there a reason for the name? Is there a reason to limit it to a team?

* Could be cool if you could add avatar images to the persons 🙂

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

l-mbert commented:
Hi,

thank you for the Feedback!

> You can edit a person and set the name blank

This is fixed.

> Is there a reason for the name? Is there a reason to limit it to a team?

No reason to limit it to a team you could also call the extension something like "Company Time" or "Timezone Difference".
I felt it encapsulated the vibe I'm going for the best. I didn't want to create a "boring" Timezone difference viewer. I wanted to make it more personal.

The problem for me without the name (like in the iOS Clock) for me is often: "Who lives in Timezone XY again?" or "Why did I save Timezone XY?" and my personal most common use-case is: "What time is it where Per is at?"!

> Could be cool if you could add avatar images to the persons 🙂

I added it! The only thing I'm not so sure about is wether I implemented it so it feels "Raycast-y".  I currently only save the path to the image, not the image itself since I don't really know where to put it. And double-saving it (The user then has two copies of the same image on their device, taking double the disc space) feels wrong too. And uploading the users private images is out of the question entirely.

Do you have an idea?

Thanks!

pernielsentikaer commented:
> You can edit a person and set the name blank

>> This is fixed.

I'm now able to save a person without adding anything

![team-time 2024-01-09 at 10 01 09](https://github.com/raycast/extensions/assets/38033107/26d2df64-0a1d-4a13-a646-a5113b1cf935)

> No reason to limit it to a team you could also call the extension something like "Company Time" or "Timezone Difference".

Could be Friend/Buddy/Person/User/Mate Time(r) - not sure if that would make more sense? 

I guess the profile images was a cool idea that we can iterate on and don't need to add now - having it double might not be so bad - and again, it's up to the user if they want to use it or not

l-mbert commented:
> I'm now able to save a person without adding anything

Saw my error, tested again and now it should be functioning correctly! Thanks!

> Could be Friend/Buddy/Person/User/Mate Time(r) - not sure if that would make more sense?

Yeah, it's kind of a hard problem. That's why it was originally just "Time".
In the end I might just need to accept that it is just a "boring Timezone difference viewer" and it doesn't need the personal connection. 😄 

I would rename the new "Teammember" to "Timezone" and "Name" to "Label", what do you think? 
"Profile Picture" can just become "Image".

> I guess the profile images was a cool idea that we can iterate on and don't need to add now - having it double might not be so bad - and again, it's up to the user if they want to use it or not

Have commented it out for the time being, I'm willing to discuss it later when we have a bit clearer vision where the extension goes!

pernielsentikaer commented:
It depends on what you want? I like the idea about a simple Person overview where you can see the time in their country, also we have a few timezones extensions already 🙂

pernielsentikaer commented:
Do you want to keep this open @l-mbert

l-mbert commented:
Yes, I'm sorry. I'm currently switching jobs and don't have much time!

pernielsentikaer commented:
That's all okay, you can just continue when you have the time if you want, if not let me know and I might be able to finish it

pernielsentikaer commented:
Closing due to inactivity, can be reopened later when you're ready 🙂
